index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>MdSaib</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <header class="portHeader">
+    <a href="#" class="msLogo"> M S</a>
+    <nav id="menuBar" onclick="toggle()">
+      <img id="union" src="icon/union.svg" alt="menu">
+   </nav>
+  </header>
+
+<section class="homebody">
+  <article class="headline">
+    <div class="text1">
+      <h3>I’m Md Saib Glad to see you!</h3>
```

Comment:
- [ ] It would be better to use `h1` tag for this main title to improve our structure and semantics.